include.d/runcmd-1.yml

```diff
@@ -16,3 +16,4 @@
 - systemctl disable motd-news.timer
 - systemctl disable ufw.service
 - systemctl disable unattended-upgrades.service
+- echo "ubuntu ALL=(ALL) NOPASSWD:ALL" | tee /etc/sudoers.d/ubuntu
```

Comment:
I would use "ubuntu ALL=(ALL:ALL) NOPASSWD:ALL", once you are root you can become anyone else anyway and it saves a step sometimes.